modules/installation-vsphere-machines.adoc

```diff
@@ -12,11 +12,26 @@ Before you install a cluster that contains user-provisioned infrastructure on VM
 .Prerequisites
 
 * Obtain the Ignition config files for your cluster.
-* Have access to an HTTP server that you can access from your computer and that the machines that you create can access.
```

Comment:
No longer hosting bootstrap Ignition config on webserver, so this prereq isn't necessary.